spec/presenters/activity_csv_presenter_spec.rb

```diff
@@ -173,6 +173,33 @@
     end
   end
 
+  describe "#linked_activity_identifier" do
+    context "when the activity is not ISPF-funded" do
+      it "returns nil" do
+        activity = build(:programme_activity, :gcrf_funded)
+
+        expect(described_class.new(activity).linked_activity_identifier).to be_nil
```

Comment:
`described_class` 👀

Reply:
Is this a "we want to avoid `described_class`" or have I got something wrong here?

Reply:
It's an "I thought you had a strong distaste for `described_class`" 😅

Reply:
I do! But I don't want to break from the norm here and I think people are still using it so I've been biting my tongue...